app/serializers/group_post_user_serializer.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
 class GroupPostUserSerializer < BasicUserSerializer
-  attributes :title, :name
+  attributes :title, :name, :primary_group_name
+
+  def primary_group_name
+    object.primary_group.name if object.primary_group
```

Comment:
Do we need to ensure the name is CSS-class-friendly?

Reply:
So I thought about that, but it appears like this is the same format as what we're doing for printing out the primary group to the post as well: https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/app/widgets/post.js#L746

I err'd on the side of consistency, and group names can't have spaces in them anyway.